docs/event-bus/docs/040-details-troubleshooting.md

```diff
@@ -21,6 +21,21 @@ document introduces several ways to troubleshoot such problems.
  If these general guidelines do not help, go to the next section of this
  document.
 
+## Search by tags
+It is possible to search the traces using tags. Tags are “key-value” pairs configured for each service.
+
+The full list of tags for a service can be seen from the details of that service's span.
```

Comment:
`See the full list of tags for a service from the details of that service's span.`